### ISSUE TITLE
Inline docstrings for attributes don't show up unless docstring is on very next line

### ISSUE THREAD

**Issue by Gobot1234** [microsoft/pylance-release#3769]
**Describe the bug**
When using inline docstrings for attributes the LSP isn't able to show documentation for attributes that aren't on the very next line after the assignment.

**Screenshots or Code**
```python
class App:
    def __init__(self):
        self.playtime_linux = ...
        """The total amount of time the user has played the app on Linux."""  # works
        self.icon = (
            ...
        )
        """The icon of the app."""  # doesn't show up
```
<img width="639" alt="image" src="https://user-images.githubusercontent.com/50501825/208327343-83b6f432-27fe-4a0c-b25e-91e94833917f.png">
<img width="770" alt="image" src="https://user-images.githubusercontent.com/50501825/208327371-1226b554-1ba1-4bff-a50c-2ee79ab44276.png">

**Expected behavior**
I expect both ways of documenting the attribute to work

**VS Code extension or command-line**
pylance: v2022.12.21

**Comment by rchiodo** [comment]
Looks like the first one works now:



![Image](https://user-images.githubusercontent.com/19672699/221294473-9634d77f-ea81-403b-9ae8-1edd1f6bb11e.png)


But the slots one doesn't:



![Image](https://user-images.githubusercontent.com/19672699/221294588-a93bc3f6-9c63-4c8f-a498-d17a1264134e.png)

**Comment by rchiodo** [comment]
For the slots problem, the hover provider would have to use the docstring from the second declaration. Right now it always uses the first which is the slots one.

Similar to what is done here:
https://github.com/microsoft/pyrx/blob/52b7ab38c2435ec0bd516f73b19697c66af6770e/packages/pyright/packages/pyright-internal/src/languageService/hoverProvider.ts#L122

**Comment by bschnurr** [comment]
This issue has been fixed in prerelease version 2023.3.11, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023311-1-March-2023-PreRelease)
